### PR TITLE
chore(lint): add some vrsource tslint rules

### LINKED ISSUE #11800: Prevent jasmine focus tests
What about switching to a tslint rule ?

- probably more robust than the current solution,
- would work on `gulp lint`

ref:  https://github.com/vrsource/vrsource-tslint-rules / "no-jasmine-focus"

/cc @pkozlowski-opensource

### PR DESCRIPTION
**Pull request by alfaproject**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[x] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?**
No TSLint rule is being used to check for Jasmine focus methods

**What is the new behavior?**
TSLint rules were added from vrsource to improve the linting task
Fixes #11800
Relates to #12548

**Does this PR introduce a breaking change?**

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: modules/@angular/examples/testing/ts/testing.ts [hunk 1]
```diff
@@ -20,17 +20,6 @@ describe('some component', () => {
 });
 // #enddocregion
 
-// #docregion fdescribe
```
Comment (vicb) [inline modules/@angular/examples/testing/ts/testing.ts]: why is this removed ? 
the file should rather be excluded

File: npm-shrinkwrap.readme.md [hunk 2]
```diff
@@ -47,5 +47,5 @@ To Remove an existing dependency do the following:
 3. ensure that your `node_modules` directory is not stale by running `npm install`
 4. run `npm uninstall --save-dev <packagename>@<version|latest>`
 5. run `./tools/npm/reshrinkwrap`
-6. these steps should change 3 files: `npm-shrinkwrap.json` and `npm-shrinkwrap.clean.json`.
+6. these steps should change 3 files: `package.json`, `npm-shrinkwrap.json` and `npm-shrinkwrap.clean.json`.
```
Comment (vicb) [inline npm-shrinkwrap.readme.md]: not sure about this change, could you expand ?

Comment (alfaproject) [inline npm-shrinkwrap.readme.md]: When you uninstall a dependency, package.json also changes

Comment (vicb) [inline npm-shrinkwrap.readme.md]: you're right, I was reading too fast.

**Comment by vicb** [comment]
Please add `fixes #11800` to the last commit message

**Comment by alfaproject** [comment]
@vicb should be clean now (;

File: modules/@angular/examples/testing/ts/testing.ts [hunk 3]
```diff
@@ -21,6 +21,7 @@ describe('some component', () => {
 // #enddocregion
 
 // #docregion fdescribe
+/* tslint:disable-next-line:no-jasmine-focus */
```
Comment (vicb) [inline modules/@angular/examples/testing/ts/testing.ts]: :+1:

File: gulpfile.js [hunk 4]
```diff
@@ -125,22 +125,8 @@ gulp.task('public-api:update', ['build.sh'], (done) => {
       .on('close', done);
 });
 
-// Checks tests for presence of ddescribe, fdescribe, fit, iit and fails the build if one of the
```
Comment (alexeagle) [inline gulpfile.js]: Bad merge?

**Comment by alexeagle** [comment]
LGTM

File: gulpfile.js (hunk 4 above)
Comment (vicb) [inline gulpfile.js]: nope, this is checked via lint now

**Comment by vsavkin** [comment]
Merged https://github.com/angular/angular/commit/e86573bac8a87787e415d1615b09f7e3d30ecb30
